_studio/mfx_lib/encode_hw/h264/include/mfx_h264_encode_hw_utils.h

```diff
@@ -853,6 +853,29 @@ namespace MfxHwH264Encode
         mfxU32 cost;
     };
 
+#if !defined(MFX_EXT_BRC_DISABLE)
+    struct BRCFrameParams : mfxBRCFrameParam
+    {
+        mfxU16 picStruct;
+    };
+#else
+    struct BRCFrameParams
+    {
+        mfxU32 reserved[23];
```

Comment:
That's not a part of public API that we need to be ABI compatible. Why do we need reserved fields here?